Tools/autotest/arducopter.py

```diff
@@ -3044,52 +3044,54 @@ def test_motortest(self, timeout=60):
         self.end_subtest("Testing percentage output")
 
     def fly_precision_sitl(self):
```

Comment:
Misnamed now.

Perhaps we should split this like we do Plane's airspeed testing  - have acommon subroutine called for each backend you want to test?

Reply:
why it is misnamed ? both are SITL only backend ! Anyway, I have change the name